src/components/DocumentationTopic.vue

```diff
@@ -55,7 +55,7 @@
           <TechnologyList
             v-if="extendsTechnology"
             class="extends-technology"
-            title="Extends"
+            :title="{ en: { one: 'Extends', other: 'Extends' } }"
```

Comment:
This line was really confusing to me at first. I second what Marcus said above about providing the ability to pass a static title that doesn't get pluralized. Maybe that means creating another Plural category? Something like 'noChange', or 'verb'? Or now that I think about it, technically this case is pluralizing 'extend'. So could we just call the pluralize function with pluralize('extend', 2)?

Reply:
> This line was really confusing to me at first. I second what Marcus said above about providing the ability to pass a static title that doesn't get pluralized.

Makes sense. Really we shouldn't even try to pluralize "Extends" in this scenario since it isn't even a noun. I made that change with 311de9e071be0c24bda6dad4e44359c522b6a19e

> Maybe that means creating another Plural category? Something like 'noChange', or 'verb'? Or now that I think about it, technically this case is pluralizing 'extend'. So could we just call the pluralize function with pluralize('extend', 2)?

Agree that we shouldn't try to pluralize that word specifically. I don't agree that we should workaround this type of situation with a new plural category that doesn't fit into the normal standard.